group-work.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: episode
+title: How to organize a group's work
+teaching: 15
+exercises: 15
+questions:
+  - Should I start a repo under my account or open a new organization?
+  - Should I add everybody as collaborator?
+  - GitHub vs GitLab?
+  - What permissions to choose?
+  - Pull requests vs direct commits for types of projects
+  - Small vs large changes
+---
+
+# How to organize a group's work
+
+---
```

Comment:
I almost never put these horizontal rules in.  What's the reason for them in markdown, do they do something for jekyll/md?

I just thought that they could be a way to select scroll points for slideshow mode...

Reply:
It sometimes helps me to visually separate them when reading but it's not necessary.